modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -550,4 +553,71 @@ private function fieldFormatSocialElement($items, $body) {
       $body->addChild($social);
     }
   }
+
+  /**
+   * Add ads if configured in settings
+   */
+  public function addAdsFromSettings() {
```

Comment:
@scottrigby Agreed. It would make sense to put this in the base class you have been working on in #35 (same thing goes for the analytics added in #34), so we should revisit there.